pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -101,4 +96,45 @@ func (backfill *Action) Execute(ssn *framework.Session) {
 	}
 }
 
+func predicateforBackfill(ssn *framework.Session, task *api.TaskInfo, node *api.NodeInfo, fe *api.FitErrors) error {
```

Comment:
Can we consolidate the `predicateforBackfill`, `predicateforReclaim`, `predicateforAllocate` and `predicateforPrempt`?

Reply:
done